### ISSUE TITLE
2.19 Cannot refresh sources: Internal Server Error

### ISSUE THREAD

**Issue by lacereation** [SSilence/selfoss#1165]
I am getting `Internal Server Error` messages since I installed 2.19.  Running    `selfoss.nukeLocalData()` in the console seemed to be truly radioactive ;).  I would get `Cannot log out: Internal Server Error`.  Then my sources and tags would disappear in #nav.  I would run `selfoss.nukeLocalData()` again and they would come back, but not work and would disappear again. I fixed that by deleting the sqlite db and symlinking the db from 2.18, which I still have installed on my server.  I still get `Cannot refresh sources: Internal Server Error` every time I refresh my sources, but do not get consistent behavior.  Sometimes it works as it should, sometimes the new items will show up when I refresh the page and sometimes I have to update from 2.18.

**Comment by jtojnar** [comment]
Did you reload the page after running `nukeLocalData()`? If the app was running old scripts, `nukeLocalData()` would clear them out of the cache but new scripts would not be used until the page was reloaded. It might even cause some Internal Server Errors when the server and client code use incompatible data structures.

To find out more, could you enable the `DEBUG` mode by setting `logger_level=DEBUG` in `config.ini` and replacing `set('DEBUG', 0)` for `set('DEBUG', 1)` in [`src/common.php`](https://github.com/SSilence/selfoss/blob/77c3f3f585a18e25f9bfccea158b6354b48d390c/src/common.php#L19) then there should be a proper backtrace in the body of the HTTP response and also in the logs (defaults to `data/logs/default.log`).

**Comment by lacereation** [comment]
Ostensibly changing those log settings--that is all I did besides messing around with user.css--wiped out my sources and tags.  So I did a nukeLocalData(), refreshed and did some ctrl-F5's for good measure. Now I have no data whatsoever, just says 'No Entries Found'.  Nothing has been written to my log file in weeks, incl now.  Tried another nuke--did nothing.  I copied it over from 2.18-- will check permissions.
